tests/core/test_broker.py

```diff
@@ -301,6 +301,7 @@ async def test_kafka_broker_security_protocols(file: Text, exception: Exception)
         actual._create_producer()
 
 
+@pytest.mark.skip_on_windows
 async def test_no_pika_logs_if_no_debug_mode(caplog: LogCaptureFixture):
```

Comment:
Yes, this is unrelated and was discussed in https://github.com/RasaHQ/rasa/pull/10994. 
As soon as https://github.com/RasaHQ/rasa/pull/10994 gets merge this will not appear in this PR anymore